examples/swarmion-full-stack/nx.json

```diff
@@ -5,9 +5,9 @@
       "runner": "nx/tasks-runners/default",
       "options": {
         "cacheableOperations": [
+          "bootstrap",
```

Comment:
can nx cache a script that is not a target ?

Reply:
Yes, it will simply not run it